integration_tests/auction_test.go

```diff
@@ -106,7 +107,10 @@ func (s *IntegrationTestSuite) TestAuctionModule() {
 		s.T().Log("Waiting for proposal to be approved..")
 		s.Require().Eventually(func() bool {
 			proposalQueryResponse, err := govQueryClient.Proposal(context.Background(), &govtypes.QueryProposalRequest{ProposalId: 1})
-			s.Require().NoError(err)
+			if err != nil {
+				time.Sleep(3)
```

Comment:
I believe `Sleep` is expecting a `Duration`, the linter indicates this will sleep for three nanoseconds. I don't think we should need a sleep at all...baked into `Eventually` is time between executions.